orangewidget/widget.py

```diff
@@ -962,6 +965,31 @@ def __updateSavedGeometry(self):
             # Note: This should always be stored as bytes and not QByteArray.
             self.savedWidgetGeometry = bytes(self.saveGeometry())
 
+    def sizeHint(self):
```

Comment:
I understand why this would be useful for graphs, but could you please explain why is this method useful if you don't have mainArea?

Also, I do not see a way to get the original Qt behavior back. Of course, I could reimplement sizeHint and return some constants, but that is something else. What you suggest might be a bad heuristic for some specific widget, therefore I'd like a way to disable it.

Reply:
Since I am asking you to make this sizeHint an option... Perhaps you could also expose the 1.1 ratio? Perhaps have an attribute defining the ratio that defaults to 1.1 and then allowing the widget to set it differently? And then, if it is False (or None, I do not know) it would just return `super().sizeHint()`?

Some context: current defaults (and fixed implementation) could be problematic for widgets with two vertically arranged graphs in the main area: I have some, Hyperspectra and Preprocess Spectra.